test/config/fixtures/mock-config-all.xml

```diff
@@ -240,6 +240,7 @@
         <accept-ogg-theora>yes</accept-ogg-theora>
         <agent command="vlc" arguments="-I dummy %in --sout #transcode{venc=ffmpeg,vcodec=mp2v,vb=4096,fps=25,aenc=ffmpeg,acodec=mpga,ab=192,samplerate=44100,channels=2}:standard{access=file,mux=ps,dst=%out} vlc://quit" />
         <buffer size="14400000" chunk-size="512000" fill-size="120000" />
+        <sample-frequency>44100</sample-frequency>
```

Comment:
This must be part of ConfigGenerator code. Otherwise the builtin default is used automatically.

Reply:
I removed it from the mock config file.